torch/csrc/utils/cuda_enabled.h

```diff
@@ -1,15 +1,13 @@
 #pragma once
 
-namespace torch {
-namespace utils {
+namespace torch::utils {
 
-static inline bool cuda_enabled() {
+inline constexpr bool cuda_enabled() {
```

Comment:
Not sure this should be constexpr as we may change it in the future to require some dynamic dispatch etc to determine if CUDA is really enabled.

Reply:
constexpr can contain non-constexpr expressions.

Reply:
Yeah, but downstream libraries could break if they depend on it being constexpr and we revert it's constexpr-ness later.